Divi/staff dashboard

didvi opened the pull request:
## Staff Dashboard

Main page for Staff to view participants

kyuwonk commented:
It seems like the merging hasn't been completely done? Or the wrong commit was selected when resolving merge conflicts?

didvi commented:
I fixed the container sizing problem and added a message for when there are no participants to show, I also believe that I fixed the font problem but I am not sure what it will look like on your computer. Let me know if it is still showing up weird.
